functions/task1.js

```diff
@@ -0,0 +1,19 @@
+// function declaration
+function squareSurface ( width, height ) {
```

Comment:
function name should be an action - "what to do". Example - `calculateArea`

Reply:
Updated the names of function.